core/designsystem/src/main/kotlin/com/naveenapps/expensemanager/core/designsystem/ui/extensions/ContextExt.kt

```diff
@@ -40,6 +44,9 @@ fun openEmailToOption(context: Context, emailId: String) {
         context.startActivity(intent)
     } catch (e: ActivityNotFoundException) {
         // Define what your app should do if no activity can handle the intent.
+        Toast.makeText(context, context.getString(R.string.no_email_app_found), Toast.LENGTH_SHORT)
```

Comment:
Hey @Tonnie-Dev, I appreciate your effort. Would returning the error message and handling it on the client/caller end make sense?

Reply:
@Tonnie-Dev Actually you are right, this is not the correct approach, I will edit my commits

Reply:
By the way have you set out [Buy Me Coffee](https://www.buymeacoffee.com/) Account, Have learnt great deal from this repo and I would like to thank you personally 🙏🏽